### ISSUE TITLE
[a11y]: Button and listbox of ComboBox component does not have tabbable element

### ISSUE THREAD

**Issue by LauKwaiFanHK** [carbon-design-system/carbon#11988]
### Package

@carbon/react

### Browser

Chrome, Firefox

### Package version

v10

### Automated testing tool and ruleset

IBM Accessibility Checker - Latest Deployment

### Description

I used IBM Accessibility Checker to scan a newly developed page based on React which uses the ComboBox component. When I scan this component independently (the page on storybook), a list of warning are reported. Please clarify if there are false positives and if there are true positives that need to be fixed. Thanks.

### Reproduction/example

Warnings are found on Carbon-React storybook for v10

### Steps to reproduce

To reproduce violations on v10:
1. Go to https://v10-react.carbondesignsystem.com/iframe.html?id=components-combobox--combobox-story&viewMode=story&args=
2. Scan this page using IBM Accessibility Checkers
3. A list of warnings will be found:
![Screenshot 2022-08-22 at 15 36 01](https://user-images.githubusercontent.com/81176980/185934199-58ea0575-e627-4d17-8efa-cf9d6ccb0586.png)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Thanks for opening this issue. These warnings do not constitute a problem. Generally the items that require a change are errors. 

Oftentimes with these warnings you can yourself validate if there is a true issue.  One warning for instance is to "Verify the <input> element with "combobox" role has keyboard access." You can tab through the component and use the arrow keys to see the combobox has full keyboard navigation support.
